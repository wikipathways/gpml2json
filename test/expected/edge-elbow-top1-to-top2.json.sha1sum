0d13edf42ac85af9745b438d05ea32c6f3b3761f ?-
