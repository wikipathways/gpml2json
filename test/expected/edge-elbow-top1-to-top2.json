{"pathway":{"@context":["https://cdn.rawgit.com/wikipathways/WpVocabularies/7a46a05/contexts/pvjs.jsonld"],"backgroundColor":"white","borderWidth":0,"color":"black","contains":["d0ffc","c2913","a165e"],"drawAs":"rect","gpmlElementName":"Pathway","height":125.99058202057536,"kaavioType":"Group","name":"test case","padding":5,"type":["Pathway"],"width":348.4903574913578,"x":0,"y":0,"zIndex":0,"fontSize":12,"fontWeight":"bold","textAlign":"left","verticalAlign":"top","dataSourceVersion":"20130813","standardName":"test case","displayName":"test case","textContent":"Name: test case","isSimilarTo":"http://wikipathways.org/index.php/Special:SearchPathways?query=test%20case&doSearch=1","id":"hash://sha256/86b7b649686fa75c0e4e084aa740ece9a28ffdee15710909946cbd3db3cee7e0"},"entityMap":{"c2913":{"id":"c2913","textContent":"GeneProduct","wpType":"GeneProduct","x":238.49035749135783,"y":52.68005543907303,"fontSize":10,"height":20,"verticalAlign":"middle","width":80,"zIndex":32969,"textAlign":"center","color":"#000000","backgroundColor":"#ffffff","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"drawAs":"rect","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["DataNode","SingleFreeNode","GeneProduct"],"kaavioType":"SingleFreeNode","gpmlElementName":"DataNode"},"a165e":{"id":"a165e","textContent":"0003","wpType":"GeneProduct","x":18.580879754708718,"y":75.99058202057536,"fontSize":10,"height":20,"verticalAlign":"middle","width":80,"zIndex":32970,"textAlign":"center","color":"#000000","backgroundColor":"#ffffff","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"drawAs":"rect","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["DataNode","SingleFreeNode","GeneProduct"],"kaavioType":"SingleFreeNode","gpmlElementName":"DataNode"},"d0ffc":{"id":"d0ffc","drawAs":"ElbowLine","borderWidth":1,"zIndex":32968,"points":[{"isAttachedTo":"a165e","attachmentDisplay":{"position":[0.5,0],"offset":[0,0]},"x":58.58087975470872,"y":75.99058202057536,"orientation":[0,-1]},{"x":178.53561862303326,"y":32.68005543907303},{"isAttachedTo":"c2913","attachmentDisplay":{"position":[0.75,0],"offset":[0,0]},"x":298.4903574913578,"y":52.68005543907303,"orientation":[0,1]}],"color":"#000000","type":["Interaction","Edge"],"kaavioType":"Edge","gpmlElementName":"Interaction","markerStart":"TBar","markerEnd":"Arrow","isAttachedTo":["a165e","c2913"]}}}
